docs/governance/templates/release_issue.md

```diff
@@ -37,6 +40,7 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] Run `git remote update && git checkout master && git reset --hard upstream/master` to ensure your code is in line with upstream  (unless this is a hotfix, then do the same, but for the release branch)
 - [ ] Publish SDK packages
    - [ ] Run `make sdk-shell-node` to get interactive shell to publish node package
+   - [ ] Run `make sdk-publish-csharp` to deploy to NuGet. (if release candidate: `make sdk-publish-csharp RELEASE_VERSION={version}-rc`) 
```

Comment:
This should say something about how/where to get a token from and that it will be prompted to be passed in.

Reply:
Added link to release notes.